Makefile

```diff
@@ -80,16 +80,23 @@ create-release-assets:
 		zip -r release/$${d}.zip $${d} ; \
 	done
 
-build-pyzq:
-	@mkdir -p dist/pybuild
-	@go build -buildmode=c-shared -o dist/pybuild/zq.so ./python/zq.go
+build-python-wheel: build-python-lib
+	pip3 wheel --no-deps -w dist ./python
+
+build-python-lib:
```

Comment:
Note: I could not find a way to have the archive live under `dist/...` and still have it usable by the pip wheel / setup.py bdist_wheel process, hence the archive is created under `python/`.